src/Extensions/Abstractions/EventIds.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Event ids enum</summary>
+	public enum EventIds : ushort
+	{
+		/// <summary>Event Id for logging general info message</summary>
+		LogInfoEventId = 1,
+
+
+		/// <summary>Event Id for logging error message</summary>
+		LogErrorEventId = 2,
+
+
+		/// <summary>Event Id for logging warning message</summary>
+		LogWarningEventId = 3,
+
+
+		/// <summary>Event Id for logging verbose message</summary>
+		LogVerboseEventId = 4,
+
+
+		/// <summary>Event Id for logging spam message</summary>
+		LogSpamEventId = 5,
+
+
+		/// <summary>Event Id for logging timed scopes</summary>
+		LogTimedScopeEventId = 6,
+
+
+		/// <summary>Event Id for logging timed scopes traces</summary>
+		LogTimedScopeTrxEventId = 7,
```

Comment:
`Trx` name is legacy (I've highlighted this in the other PR to TimedScopes) We should change it like below:
```csharp
 /// <summary>
 /// Event Id for test context timed scopes; ex: health checks, observers
 /// </summary>
 LogTimedScopeTestContextEventId = 7,
```

Reply:
Fixed, thank you